src/Presentational/BeneficiaryCard.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import "../index.css";
+
+const Tag = ({tags}) => (
+  <div>
+    {tags.map(tag => (
+      <div>{tag}</div>
+    ))}
+  </div>
+);
+
+export default class BeneficiaryCard extends Component {
+  render() {
+    return (
+      <div className="needCard"> 
+        <label className="needCard title">{this.props.name}</label>
+        <label className="needCard content">¿Dónde?</label>
+        <label className="needCard content">
+          <a href="{this.props.mapsurl}">{this.props.address}</a>
```

Comment:
`<a href={this.props.mapsurl}>...</a>` Sin las `"`. Si este link es dentro de la app entonces hay que usar el `<Link>` de `react-router`